config/localmodelnodes/manager.yaml

```diff
@@ -21,6 +21,8 @@ spec:
       annotations:
         kubectl.kubernetes.io/default-container: manager
     spec:
+      nodeSelector:
```

Comment:
Can this be configured ?

Reply:
updated it, but I haven't test it yet. 😅